# PDGWAS_regulatorynetworks

This repository contains the supplementary tables for the paper "Establishing gene regulatory networks from Parkinson's disease risk loci"

Supplementary table 1: Nalls et al. 2019 GWAS dataset. Data downloaded directly from the publication Supplementary files. Odds ratios obtained from the IPDGC PD locus web browser. 
Supplementary table 2: The Hi-C datasets used in CoDeS3D analysis. List of all Hi-C datasets used in this study and their reference to original article and GEO accession number
Supplementary table 3: Nalls et al. 2019 GWAS: Sig eQTLs CoDeS3D output. Detailed data of eQTL-gene interactions when interrogating the 90 PD-SNPs. Overall 76 of the 90 SNPs tested were identified as eQTLs that were associated with the regulation of 518 genes across 49 tissues. 
Supplementary table 4: Nalls et al. 2019 GWAS: Sig eQTLs CoDeS3D output: Brain Hi-C: Brain GTEx only. Detailed data of eQTL-gene interactions when interrogating the 90 PD-SNPs in the 13 GTEx brain tissues only, and using brain-specific Hi-C cell lines (cortical plate neurons; germinal zone neurons; astrocyte of the cerebellum; brain vascular pericyte; brain microvascular endothelial primary cell; SK-N-MC; SK-N-DZ; neuronal progenitor cells; astrocyte of the spinal cord; dorsolateral prefrontal cortex cells; hippocampus cells; H1 neuronal progenitor cells) 
Supplementary table 5: SNPnexus epigenome roadmap for all 90 SNPs, including histone modifications and DNaseI marker of open chromatin
Supplementary table 6: Pathway analysis of the 518 genes; gProfiler (g:OST)
Supplementary table 7: The LOEUF values (downloaded from gnomAD) for each of the 518 genes included in the gene network. LOEUF score ("loss-of-function observed/expected upper bound fraction”) indicates the tolerance of a given gene to inactivation. A low LOEUF score indicates strong selection against loss-of-function variation.
Supplementary table 9: Protein:protein interaction analysis of 523 genes (using STRING, high confidence levels >0.700) followed by Louvain clustering identified 9 significant clusters. 
Supplementary table 10: Pathway analysis for the genes within each of the nine significant clusters; gProfiler (g:OST) results
Supplementary table 11: Sig eQTLs CoDeS3D output for rs10835060 and rs4238361: These two SNPs were identified by Makarious et al. as part of a polygenic risk score for PD diagnosis
Supplementary table 12: Correlational analysis – tissues falling outside of the 95% CI of expected eQTL numbers. Detailed data of the tissues that fall above or below the 95% CI of expected eQTL numbers, supplemental to figure 2. The data includes information for the analysis run on all 49 tissues (i.e. figure 2e-h). 
